data/admin/ingredient.py

```diff
@@ -52,3 +52,20 @@ class IngredientAdmin(ElementAdminWithChangeReason):
 
     def has_linked_substances(self, obj):
         return "Oui" if obj.substances.exists() else "Non"
+
+    def save_model(self, request, obj, form, change):
```

Comment:
Je n'était pas très sûre de l'ordre dans lequel les appels à `save()` seraient résolus.
Mais il semble, selon https://stackoverflow.com/questions/24292847/does-it-make-sense-to-override-the-save-method-in-a-mixin
* appel à save_model() de XAdmin
* puis appel à save_model de ElementAdminWithChangeReason